package.json

```
@@ -1524,6 +1524,18 @@
                         "required": [
                             "appProject"
                         ]
+                    },
+                    "dockerRun": {
+                        "description": "%vscode-docker.tasks.dotnet-container-sdk.dotnetContainerSdk.description%",
```

```suggestion
                        "description": "%vscode-docker.tasks.dotnet-container-sdk.dockerRun.description%",
```

The package.json property is still `dockerRun`, so I'd leave that part of the ID unchanged.

---

that's right